src/activity-logger/background/log-page-visit.js

```diff
@@ -23,26 +24,32 @@ async function storeVisit({ timestamp, url, page }) {
  *  to the new page.
  * @param {any} visit The new visit to add to index; should occur regardless of reidentify outcome.
  */
-async function updateIndex(finalPagePromise, visit) {
-    // Wait until all page analyis is done
-    const { page } = await finalPagePromise
+async function updateIndex(storePageResult, visit, pageId) {
+    if (storePageResult == null) {
+        index.addMetaConcurrent('visit', Date.now(), pageId)
+    } else {
+        // Wait until all page analyis is done
+        const { page } = await storePageResult.finalPagePromise
 
-    // If no page returned from analysis, we can't index
-    if (!page) {
-        return
-    }
+        // If no page returned from analysis, we can't index
+        if (!page) {
+            return
+        }
 
-    // Index either the page + visit, or just the visit if page already exists
-    try {
-        await index.addPageConcurrent({ pageDoc: page, visitDocs: [visit] })
-    } catch (error) {
-        // Indexing issue; log it for now
-        console.error(error)
-        throw error
+        // Index either the page + visit, or just the visit if page already exists
```

Comment:
This comment is a bit out of date now. Could lead to confusion. Either remove or update, and maybe add another comment in the other condition (`if (storePageResult == null)`) to explain what's going on in here